tests/elements-regression/src/pages/editor-page.js

```diff
@@ -83,15 +83,44 @@ module.exports = class EditorPage extends BasePage {
 		return await this.addElement( { widgetType, elType: 'widget' }, container );
 	}
 
+	/**
+	 * @return {import('@playwright/test').Frame}
+	 */
 	getPreviewFrame() {
 		return this.page.frame( { name: 'elementor-preview-iframe' } );
 	}
 
+	async waitForElementRender( id ) {
```

Comment:
Try to wait for an event like `finalRender`

Reply:
Tried this one, and it feels like a mess, we need to do so, but maybe it should be in another task.